preupg/kickstart/plugins/partitioning.py

```diff
@@ -182,21 +202,29 @@ def get_volume_info(filename, first_index, second_index):
     def get_partition_layout(self, lsblk, vgs, lvdisplay):
         """
         Returns dictionary with partition and realname and size
-        :param filename:  filename with partition_layout in /root/preupgrade/kickstart directory
+        :param filename:  filename with partition_layout in
+                          /root/preupgrade/kickstart directory
```

Comment:
Most probably isn't :) I changed it to have each line less then 80 chars, but you're right, I'll update the content to reflect the current params.